docs/UsersGuide/source/ConfigWorkflow.rst

```diff
@@ -130,15 +130,6 @@ METplus Parameters
 ``MODEL``: (Default: "")
    A descriptive name of the user's choice for the model being verified.
    
-``MET_INSTALL_DIR``: (Default: "")
-   Path to top-level directory of MET installation.
-
-``METPLUS_PATH``: (Default: "")
-   Path to top-level directory of METplus installation.
-
-``MET_BIN_EXEC``: (Default: "")
-   Name of subdirectory where METplus executables are installed.
-
```

Comment:
I know this might be opening up a can of worms, but will the removal of these variables make it harder for people to run the app using MET on unsupported platforms?

Reply:
Or is the MET package going to be released with spack-stack? If so that would be great!

Reply:
Met 10.1.1 is included in spack-stack version 1.4.1.

Reply:
The environment variables "met_ROOT" and "MET_BASE" are set by the module for met.

Reply:
@mkavulich @mark-a-potts -
The variables MET_INSTALL_DIR,and MET_BIN_EXEC are set in the modulefile for met/10.1.2, in addition to the MET_ROOT and MET_BASE. They were added because of their extensive use throughout the scripts and code. My initial attempt was indeed to keep only `MET_ROOT` and `MET_BASE` in the modulefile, and to define MET_INSTALL_DIR and  MET_BIN_EXEC in the scripts, but this needs to be done repeatedly.  The documentation for the met also suggest use for MET_INSTALL_DIR env. variable (see e.g., https://met.readthedocs.io/en/main_v10.1/Users_Guide/plotting.html)

And METPLUS_PATH is set in the modulepath for the metplus/4.1.3.